src/3d/qgs3dutils.h

```diff
@@ -363,6 +363,16 @@ class _3D_EXPORT Qgs3DUtils
      * \since QGIS 3.44
      */
     static QgsPoint screenPointToMapCoordinates( const QPoint &screenPoint, QSize size, const QgsCameraController *cameraController, const Qgs3DMapSettings *mapSettings );
+
+    /**
+     * Returns list of 4 planes, which are calculated from line defined by \a point1 and \a point2. \a distance defines
+     * the distance between parallel clipping planes and the line.
+     * Clipping planes are returned back in the format of 4D Vector, where first 3 coordinates are the coordinates of
+     * normalized normal of given plane and the last is the distance from origin of the scene.
```

Comment:
I find the wording hard to understand. Could you please rephrase it. Maybe something like: 

```
Returns a list of 4 planes derived from a line extending from \a startPoint to \a endPoint. The parameter \a distance defines the distance between the parallel clipping planes and the line.

Each clipping plane is represented as a 4D vector, where the first three components correspond to the normalized normal of the plane, and the fourth component represents its distance from the origin of the scene.
```